src/frontend/ui/entité-juridique/PageEntitéJuridique.test.tsx

```diff
@@ -181,7 +165,7 @@ describe('La page Entité Juridique', () => {
       // THEN
       const ficheDIdentité = screen.getByRole('region', { name: wording.TITRE_BLOC_IDENTITÉ })
       const majEtSource = within(ficheDIdentité).getAllByText('À venir')
```

Comment:
```suggestion
      const àVenir = within(ficheDIdentité).getAllByText('À venir')
```
Comme pour les ET MS